lib/grpc/client/adapters/mint/connection_process/connection_process.ex

```diff
@@ -0,0 +1,390 @@
+defmodule GRPC.Client.Adapters.Mint.ConnectionProcess do
+  @moduledoc """
+  This module is responsible for manage a connection with a grpc server.
+  It's also responsible for manage requests, which also includes check for the
```

Comment:
```suggestion
  It's also responsible for managing requests, which also includes checks for the
```

Reply:
Fixed in https://github.com/beligante/grpc/pull/1